jena-arq/src/main/java/org/apache/jena/riot/lang/LangJSONLD11.java

```diff
@@ -178,4 +184,22 @@ else if ( k.startsWith("@") )
             }
         });
     }
+    
+    private static final String SYMBOLS_NS = "http://jena.apache.org/riot/jsonld#" ;
+    private static Symbol createSymbol(String localName) {
+        return Symbol.create(SYMBOLS_NS + localName);
+    }
+    
+    /** Get the (jsonld) options from the jena context if exists or create default */
+    private static JsonLdOptions getJsonLdOptions(String baseURI, Context jenaContext) {
+        JsonLdOptions opts = null;
+        if (jenaContext != null) {
+            opts = (JsonLdOptions) jenaContext.get(JSONLD_OPTIONS);
```

Comment:
context.get wil do the casting (it's a generic function).

```
    JsonLdOptions opts = jenaContext.get(JSONLD_OPTIONS);
    return (opts != null) ? opts : new JsonLdOptions();
```
of an if statement if you prefer.